realm/realm-library/src/main/java/io/realm/RealmList.java

```diff
@@ -381,6 +381,24 @@ public int size() {
         }
     }
 
+    @Override
```

Comment:
Java doc needed. Especially we need to say if an object is detached from Realm, this function will always return false.